hs/src/Reach/Debugger.hs

```diff
@@ -5,15 +5,16 @@ module Reach.Debugger where
 import Control.Monad.Reader
 import Data.IORef
 import qualified Data.Map.Strict as M
+import Data.Natural
 import Reach.AST.Base
 import Reach.AST.DLBase
 import Reach.AST.LL
 
 type ConsensusEnv = M.Map DLVar DLVal
 
-type Balance = Integer
+type Balance = Natural
 
-type Token = Integer
+type Token = Natural
```

Comment:
I know I said `Nat`, but that's just because I'm an idiot. `Integer` is fine... we don't need to add more types. Sorry